exploration/open-close-expressions.md

```diff
@@ -220,3 +220,68 @@ Rendered as React this could become:
 _What other solutions are available?_
 _How do they compare against the requirements?_
 _What other properties they have?_
+
+### Do Nothing
+
+We could not provide any special support for spans and markup elements,
+and instead delegate the problem completely to tools and downstream processing layers.
+
+```
+This is <html:strong>bold</html:strong> and this is <html:img alt="an image">.
+```
+
+#### Pros:
+
+* No work required from us right now.
```

Comment:
```suggestion
* No work required from us right now. We can always add support later.
* We already support (and are required to support) in-line literal markup in messages, since they are just character sequences.

Reply:
Thanks for adding these. I agree with your last point. Re. the first two:

* "add support later" -- we'd need to carefully reserve syntax for the future addition and we'd need to do it now.
* "we already support" -- see my other comment from today; I wouldn't call it "support"; instead, perhaps "not impede" is a better phrasing.

Reply:
I'm going to commit this suggestion and propose a slightly different wording in a follow-up comment.